site.js

```diff
@@ -61,26 +87,32 @@ geocoder.on('select', function(data) {
     if (searchMarker) {
         map.removeLayer(searchMarker);
     }
-    this._closeIfOpen();
+    this._closeIfOpen(); //'this' is the geocoder search bar on the page
     searchMarker = L.marker(data.feature.center.reverse());
     searchMarker.bindPopup(data.feature.place_name);
     searchMarker.addTo(map);
 });
 
+
+/**
+* Onload callbacks for buffers and bikelanes
+*/
+
+//Styles the buffer using bufferStyle variable, and 
+function loadBuffer(data) {
+    var buffer = data.target;
+    setProperties(buffer.getGeoJSON());
+    //buffer.setGeoJSON(buffer.getGeoJSON())
+}
+
 // Each buffer feature object needs to have the properties set individually
 function setProperties(buffer) {
     for (var i = 0; i < buffer.features.length; i++) {
         buffer.features[i].properties = bufferStyle;
     }
 }
 
-// onload callbacks for buffers and bikelanes
```

Comment:
I checked out your branch locally, and started up a local web server. Removing lines 77-83 changed the colors of the buffer from blue (part of WABA's branding in https://github.com/dcfemtech/hackforgood-waba-map/issues/4) to gray. We should add these lines back.

Screenshot of https://github.com/dcfemtech/hackforgood-waba-map/commit/4127f2ba2545b2cc32d1a8828a8592fbcd806f4f on `master`:
<img width="787" alt="screenshot 2016-07-20 15 41 19" src="https://cloud.githubusercontent.com/assets/10409657/17000464/aca8e684-4e90-11e6-8fb6-ecca4bd55caf.png">

Screenshot of https://github.com/dcfemtech/hackforgood-waba-map/pull/60/commits/2aeaa1f98b02b775bdc6b8a172ac9370b9e8b2a6 on `code-comments-july`:
<img width="780" alt="screenshot 2016-07-20 15 42 04" src="https://cloud.githubusercontent.com/assets/10409657/17000488/cbf30ace-4e90-11e6-8243-fab078b2c1d0.png">